lib/jsdom/living/svg/SVGStringList.webidl

```diff
@@ -0,0 +1,14 @@
+interface SVGStringList {
+
```

Comment:
Here and elsewhere there's an extra empty newline after the opening curly that seems weird.

Reply:
It's defined this way in the SVG spec, so *shrug*.